### PR TITLE
153136675 info text

### PR DESCRIPTION
**Pull request by rckopteri**
https://www.pivotaltracker.com/story/show/153136675
https://www.pivotaltracker.com/story/show/153694814
https://www.pivotaltracker.com/story/show/153694665

### CODE REVIEW

File: ote/resources/public/language/en.edn [hunk 1]
```diff
@@ -225,7 +225,8 @@
 
   :brokerage? "Jos kyseessä on Liikenteen turvallisuusvirasto Trafille ilmoitettava välityspalvelu, laita alle raksi."
 
-  :operation-area "Add the service operation area (for example province) by typing the name in the text field and selecting it from the list. You can also draw the area with the map drawing tools. You can name the area freely (like \"Itäkeskus taxi station\")."
+  :operation-area "Lisää toiminta-alue kirjoittamalla alueen tai kohteen nimi \"Hae paikka nimellä\" -tekstikenttään ja valitsemalla haluttu alue avutuvalta listalta (tällä hetkellä haettavissa maakunta, kunta, postinumero, Suomi). Voit samalla tavalla lisätä useita alueita.
```
Comment (K1ll3rF0x) [inline ote/resources/public/language/en.edn]: Fix typo: "avutuvalta" -> "avautuvalta"

File: ote/resources/public/language/en.edn [hunk 2]
```diff
@@ -664,11 +665,11 @@
           "license-url" "License/terms of use URL"
           "external-interface" "External interface"
           "accessibility-info-url" "Accessibility description (WWW)"
-          "guaranteed-accessibility-description" "Guaranteed accessibility description"
+          "guaranteed-accessibility-description" "Kerro tarvittaessa lisätietoja aina saatavilla olevista esteettömyyspalveluista"
           "guaranteed-info-service-accessibility" "Guaranteed info service accessibility"
           "guaranteed-transportable-aid" "Guaranteed transportable aid"
           "guaranteed-vehicle-accessibility" "Guaranteed vehicle accessibility"
-          "limited-accessibility-description" "Limited availability accessibility description"
+          "limited-accessibility-description" "Kerro tarvittaessa lisätietoja rajoitetusti saatavilla olevista esteettömyyspalveluista"
```
Comment (K1ll3rF0x) [inline ote/resources/public/language/en.edn]: This input header is so long, that is getting on the input field:
image
I think we should increase the initial rows amount of free text fields to be bigger than one row.

File: ote/resources/public/language/en.edn (hunk 1 above)
Comment (rckopteri) [reply ote/resources/public/language/en.edn]: yes, thank you

File: ote/resources/public/language/en.edn (hunk 2 above)
Comment (rckopteri) [reply ote/resources/public/language/en.edn]: I leave these ui issues to Jussi

File: ote/resources/public/language/en.edn [hunk 3]
```diff
@@ -225,7 +225,8 @@
 
   :brokerage? "Jos kyseessä on Liikenteen turvallisuusvirasto Trafille ilmoitettava välityspalvelu, laita alle raksi."
 
-  :operation-area "Add the service operation area (for example province) by typing the name in the text field and selecting it from the list. You can also draw the area with the map drawing tools. You can name the area freely (like \"Itäkeskus taxi station\")."
+  :operation-area "Lisää toiminta-alue kirjoittamalla alueen tai kohteen nimi \"Hae paikka nimellä\" -tekstikenttään ja valitsemalla haluttu alue avautuvalta listalta (tällä hetkellä haettavissa maakunta, kunta, postinumero, Suomi). Voit samalla tavalla lisätä useita alueita.
```
Comment (tatut) [inline ote/resources/public/language/en.edn]: Why replace English translated text with Finnish? Translate message. (same for all messages in en.edn file)

Comment (rckopteri) [reply ote/resources/public/language/en.edn]: Koska teksti muuttuu ja meillä ei taida olla vielä missään virallista englantia?
Otanko kaikki muutokset pois tästä tiedostosta?

**Pull request merged**